File: src/JavaScript/easy/math/problem1837.js
/**
 * @param {number} n
 * @param {number} k
 * @return {number}
 */
var sumBase = function(n, k) {
    let basedNum = n.toString(k)
    let sum = 0
    for(let num of basedNum.split("")){
        sum += +num
    }
    return sum
};

// let n = 34, k = 6
let n = 10, k = 10

sumBase(n, k)

File: src/Python/easy/math/problem1837.py

# First solution

class Solution:
    def sumBase(self, n: int, k: int) -> int:
        sum = 0
        while True:
            q = n % k
            n = int(n / k)
            if n == 0 and q == 0:
                break
            sum += q
        return sum


# Second solution

class Solution:
    def sumBase(self, n: int, k: int) -> int:
        sum = 0
        while n > 0:
            sum += n % k
            n = n // k
        return sum


n = 34; k = 6

solution = Solution()
solution.sumBase(n, k)